src/Collections/Sequences/Seq.dfy

```diff
@@ -873,19 +882,19 @@ module Seq {
   ***********************************************************/
 
 
-  predicate method SortedBy<T>(a: seq<T>)
+  predicate method SortedBy<T>(a: seq<T>,compare: (T, T) -> Comparison.CompResult)
```

Comment:
```suggestion
  predicate method SortedWith<T>(a: seq<T>, compare: (T, T) -> Comparison.CompResult)
```

Reply:
Why the preference for `With` instead of `By` ?